app/controllers/api/v1/twilio_controller.rb

```diff
@@ -3,13 +3,47 @@
 
 module Api::V1
   class TwilioController < Api::V1::ApiController
-    include Webhookable
+    include TwilioConfig
 
-    after_filter :set_header, except: [:assignment, :hold_music]
+    after_filter :set_header, except: [:assignment, :hold_music, :twilio_generate_call_token]
     skip_authorization_check
     skip_before_action :validate_token
     skip_before_action :verify_authenticity_token
 
+    # OUTBOUND CALL : START
+    def connect_outbound_call
+      caller_id, offer_id, mobile = params["phone_number"].split("#")
+      redis.hmset("twilio_outbound_#{mobile}",
```

Comment:
@swatijadhav  I think it would be good to encapsulate the redis hmset/hmget calls into a service class. This would remove the underlying access and key implementation (twilio_outbound_#{mobile}) from the controller and mean that it would be easy to swap out with a different scheme later (say if we no longer wanted Redis or wanted to adjust the namespacing slightly)